public/data/nirbhay12345.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "Nirbhay Chaplot",
+    "type": "personal",
+    "bio": "Open Source Enthusiast!",
+    "avatar": "https://github.com/nirbhay12345.png",
+    "links": [
+        {
+            "name": "Follow me on GitHub",
+            "url": "https://github.com/nirbhay12345",
+            "icon": "github"
+        },
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/Nirbhay00704167",
+            "icon": "twitter"
+        }
+    ]
+}

```

Not formatted correct.

```suggestion
{
  "name": "Nirbhay Chaplot",
  "type": "personal",
  "bio": "Open Source Enthusiast!",
  "avatar": "https://github.com/nirbhay12345.png",
  "links": [
    {
      "name": "Follow me on GitHub",
      "url": "https://github.com/nirbhay12345",
      "icon": "github"
    },
    {
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/Nirbhay00704167",
      "icon": "twitter"
    }
  ]
}

```

---

Great catch, thanks! Looks like the bot has already done that for us :) Check https://github.com/EddieHubCommunity/LinkFree/commit/6e8e6621836caf631eaa76d6a93a4e65cdae2d98

---

Thank you @krishguptadev for spotting the misalignment. Will keep in mind next time